src/utils/generate-vdm-util.ts

```diff
@@ -0,0 +1,49 @@
+/*!
+ * Copyright (c) 2019 SAP SE or an SAP affiliate company. All rights reserved.
+ */
+
+import { flags } from '@oclif/command';
+import { AlphabetLowercase, AlphabetUppercase } from '@oclif/parser/lib/alphabet';
+import { IBooleanFlag, IOptionFlag } from '@oclif/parser/lib/flags';
+import { generatorOptionsCli as generatorOptionsSDK, GeneratorOptions } from '@sap/cloud-sdk-generator';
+import * as path from 'path';
+import { Options } from 'yargs';
+import { FlagsParsed } from '../commands/generate-vdm';
+
+export function toGeneratorSDK(flags: FlagsParsed): GeneratorOptions {
+  const result =  Object.entries(flags)
+    .filter(([key, value]) => typeof value !== 'undefined' && generatorOptionsSDK.hasOwnProperty(key))
+    .reduce((prev, [key, value]) => ({ ...prev, [key]: value }), {}) as GeneratorOptions;
+  return result;
```

Comment:
Why not return the return value of the reduce directly?

Reply:
eh, again I did this for debugging and did not put it back....